### PR TITLE
CHANGELOG: add missing items

### PR DESCRIPTION
**Pull request by benma**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,9 @@
 - Bundle BitBox02 v9.5.0 firmware
 - Run BitBoxApp installer as admin by default on Windows
 - Close a running BitBoxApp instance on Windows when installing an update to ensure success
+- Show connection error in detail
```
Comment (x1ddos) [inline CHANGELOG.md:9]: Which connection? USB, exchange rates, Tor proxy, ElectrumX nodes or some other connection?...

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,9 @@
...
+- Connect default BTC/LTC full nodes on port 443 to work around firewalls blocking the 5XXXXX ports
```
Comment (x1ddos) [inline CHANGELOG.md:10]: One off too many Xs :) A port like 512345 is invalid.

**Review by x1ddos — APPROVED** [review]
LGTM but please read the comments.

File: CHANGELOG.md (hunk 2 above)
Comment (benma) [reply CHANGELOG.md:10]: Damn, again I merged based on the approve checkmark :O Fixed in a new PR https://github.com/digitalbitbox/bitbox-wallet-app/pull/1224

**Pull request merged**